packages/ui/turbo.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "inputs": ["src/**"],
+      "outputs": ["dist/**"]
```

Comment:
The `theme-builder/package.json` file was getting included as a cache output from the root turbo config for some reason, so I created the nested one here to remove that.